expr/funcs.go

```diff
@@ -2,36 +2,25 @@ package expr
 
 import "github.com/raintank/metrictank/api/models"
 
-type argType uint8
-
-// argument types. to let functions describe their inputs and outputs
-// potentially not as strict as reality (e.g. movingAverage windowsize is categorized as a str) that's why we have the extra validation step
-const (
-	series      argType = iota // a single series
-	seriesList                 // a list of series
-	seriesLists                // one or multiple seriesLists
-	integer                    // number without decimals
-	float                      // number potentially with decimals
-	str                        // string
-)
-
 type Func interface {
```

Comment:
this still needs to be renamed.  You already refer to these functions as GraphiteFuncs https://github.com/raintank/metrictank/blob/12830b0b3b025bc5f4e66e7252ddab9c73fe4637/expr/types.go#L4